src/screens/Settings/index.tsx

```diff
@@ -234,14 +235,20 @@ function Settings() {
       setDefaultWinePrefix(config.defaultWinePrefix)
 
       if (!isDefault) {
+        let newInfo = await getGameInfo(appName, 'legendary')
```

Comment:
Would not be better to check the runner instead of getting the epic info and then if nothing comes check for the gog one ?